packages/opentelemetry-basic-tracer/src/Span.ts

```diff
@@ -65,6 +68,7 @@ export class Span implements types.Span {
     }
     this._kind = options.kind || types.SpanKind.INTERNAL;
     this._startTime = options.startTime || performance.now();
+    spanProcessor.onStart(this);
```

Comment:
if we already have the reference to the `tracer` should we do `this._tracer.spanProcessor(this)`?

Reply:
`_spanProcessor` is a private member in the tracer class. Need to make it public in order to achieve this.